packages/shared/ReactSymbols.js

```diff
@@ -36,6 +36,9 @@ export const REACT_CONTEXT_TYPE = hasSymbol
 export const REACT_ASYNC_MODE_TYPE = hasSymbol
   ? Symbol.for('react.async_mode')
   : 0xeacf;
+export const REACT_USE_REF_TYPE = hasSymbol
+  ? Symbol.for('react.use_ref')
```

Comment:
Let's rename these too?

Reply:
Yup. I kind of halted my renaming in the middle because of the `ReactChildFiber` suggestion I'm trying out.